pages/bills/[id]/edit.js

```diff
@@ -0,0 +1,45 @@
+import BillForm from "@/components/Form";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import styled from "styled-components";
+
+export default function EditBillPage() {
+  const router = useRouter();
+  const { id } = router.query;
+  const { isReady } = router;
+
+  const { data: bill, isLoading, error } = useSWR(`/api/bills/${id}`);
+
+  async function editBill(bill) {
+    await fetch(`/api/bills/${id}`, {
+      method: "PATCH",
+      body: JSON.stringify(bill),
+    });
+    router.push("/");
+  }
+  if (!isReady || isLoading || error) return <h2>Loading Bill..</h2>;
+
+  return (
+    <>
+      <StyledLink href={`/bills/${id}`}>X</StyledLink>
+      <Title>Edit Bill</Title>
+      <BillForm isEditPage={true} addBill={editBill} bill={bill} />
```

Comment:
your prop is called addBill but this is no longer the case. You should choose a more general name something like `onSubmit`, since it is simply the function that is executed "on submitting the form"